bundle-ocp/manifests/das-operator.clusterserviceversion.yaml

```diff
@@ -56,55 +56,52 @@ spec:
-    accelerators provided by vendors need partition to be created at node boot-time
-    or to change partitions one would have to evict all the workloads at the node
-    level to create new set of partitions.\n\nDynamic Accelerator Slicer will help
-    if:\n\n- user does not know all the accelerators partitions needed a priori on
-    every node on the cluster\n\n- user partition requirements change at the workload
-    level rather than the node level\n\n- user does not want to learn or use new API
-    to request accelerators slices\n\n- user prefers to use stable device plugins
-    APIs for creating partitions\n\n###Features Overview\n\n- Integration with Kubernetes
-    [quota management](https://github.com/openshift/instaslice-operator/blob/main/docs/instaslice_kube_quota_int.md)\n\n-
-    Integration with project [Kueue](https://github.com/openshift/instaslice-operator/blob/main/docs/kueue.md)\n\n-
-    [Emulator](https://github.com/openshift/instaslice-operator/blob/main/docs/emulator.md)
-    mode to run test Dynamic Accelerator Slicer firstfit placement strategy\n\n- Integration
-    with vLLM, Kserve, [Deployments](https://github.com/openshift/instaslice-operator/blob/main/samples/vllm_deployment.yaml),
-    [Jobs](https://github.com/openshift/instaslice-operator/blob/main/samples/vllm_job.yaml),
-    and [Statefulsets](https://github.com/openshift/instaslice-operator/blob/main/samples/vllm_statefulset.yaml)\n\n###
-    Documentation\nDocumentation and installation guide can be found below:\n  * [Installation
-    Guide](https://github.com/openshift/instaslice-operator)\n  * [Dynamic Accelerator
-    Slicer Operator](https://github.com/openshift/instaslice-operator/blob/main/README.md)\n\n###
-    License\ndas-operator is licensed under the Apache 2.0 license"
+  description: |-
+    ### Introduction
+    Dynamic Accelerator Slicer uses stable APIs and works with the NVIDIA GPU Operator to create MIG slices on demand.
+
+    ### Why Dynamic Accelerator Slicer
+
+    Partitionable accelerators provided by vendors need partition to be created at node boot-time or to change partitions one would have to evict all the workloads at the node level to create new set of partitions.
+
+    Dynamic Accelerator Slicer will help if:
+
+    - user does not know all the accelerators partitions needed a priori on every node on the cluster
+    - user partition requirements change at the workload level rather than the node level
+    - user does not want to learn or use new API to request accelerators slices
+    - user prefers to use stable device plugins APIs for creating partitions
+
+    ### Features
+
+    - Integration with Kubernetes [quota management](https://github.com/openshift/instaslice-operator/blob/main/docs/instaslice_kube_quota_int.md)
+    - Integration with project [Kueue](https://github.com/openshift/instaslice-operator/blob/main/docs/kueue.md)
```

Comment:
We don't have an integration with Kueue I thought?

Reply:
Arghhh, I forgot to remove it from the base csv, got it got generated again. Thanks. Fixed in https://github.com/openshift/instaslice-operator/pull/958